File: billing_ui_app/src/app/core/models/billing/claim-validation-model.ts
import { IdWithUserInfo, UserInfo } from "./get-claim-by-identifier";

export class ClaimValidationModel extends IdWithUserInfo {
    isSecondary : boolean;
    secondaryFunderId : number;
}

export class ClaimResponseModel{
    claimid: number;
    error?: string = '';
}

File: Microservices/BillingService/BillingService.Domain/Models/Claims/ClaimValidationModel.cs
using BillingService.Domain.Models.PaymentPosting;

namespace BillingService.Domain.Models.Claims
{
    public class ClaimValidationModel : IdWithUserInfo
    {
        public bool IsSecondary { get; set; } = false;
        public int? SecondaryFunderId { get; set; }
    }
}
